### ISSUE TITLE
Alert tone when Card Message is shown

### ISSUE THREAD

**Issue by annjawn** [SwiftKickMobile/SwiftMessages#168]
Is there a way to implement audio feedback like a message tone (just like a push notification or text message) when a message is shown?

**Comment by wtmoose** [comment]
Sure, there are callbacks where you can do what you need:

````swift
// Specify one or more event listeners to respond to show and hide events.
config.eventListeners.append() { event in
    if case . willShow = event { print("yep") }
}
````

**Comment by annjawn** [comment]
@wtmoose awesome. Thanks a lot. and great library 👍

**Comment by wtmoose** [comment]
Released in 4.1.2
